### ISSUE TITLE
NAPSI Labels missing

### ISSUE THREAD

**Issue by lukasfolle** [KDerm/NAPSI_calculator#1]
Hey there, thanks for releasing the dataset of the paper. Could you please also provide the labels generated using the NAPSI?
Best,
Lukas

**Comment by KDerm** [comment]
Thank you for the comment. The labels of ten validation sets generated after the training in Step 1 were uploaded. However, we are sorry that the test set's images and labels are not publicly available due to the form of consent.
Best,
Hiroto Horikawa

**Comment by lukasfolle** [comment]
I am talking about the labels of step 2.

**Comment by KDerm** [comment]
Sorry for the misunderstanding. The 2939 NAPSI labels for Step 2 have been uploaded.
Best,
Hiroto Horikawa

**Comment by lukasfolle** [comment]
Awesome, that’s great to hear! I missed the change of the files. Thanks for adding it!
